cmd/eip_update.go

```diff
@@ -53,16 +53,26 @@ var eipUpdateCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
+		tlsSni, err := cmd.Flags().GetString("healthcheck-tls-sni")
```

Comment:
Per convention we maintain acronyms in uppercase (when they're not prefixing the name, as it would export them out of the Go package):

```suggestion
		tlsSNI, err := cmd.Flags().GetString("healthcheck-tls-sni")
```